From: Huang Ying <ying.huang@intel.com>
Subject: x86, mce: Add HW_ERR printk prefix for hardware error logging
References: fate#311968
Git-commit: c6de9f08912311ddc1b3502b90e10fd449acd401
Patch-mainline: v2.6.36-rc1


Signed-off-by: Thomas Renninger <trenn@suse.de>

This makes hardware error related log in printk log more explicit. So
that the users can report it to hardware vendor instead of LKML or
software vendor.

Signed-off-by: Huang Ying <ying.huang@intel.com>
Reviewed-by: Hidetoshi Seto <seto.hidetoshi@jp.fujitsu.com>
LKML-Reference: <1275295689.3444.462.camel@yhuang-dev.sh.intel.com>
Signed-off-by: H. Peter Anvin <hpa@zytor.com>

---
 include/linux/kernel.h |    7 +++++++
 1 file changed, 7 insertions(+)

Index: linux-2.6.32-SLE11-SP1/include/linux/kernel.h
===================================================================
--- linux-2.6.32-SLE11-SP1.orig/include/linux/kernel.h
+++ linux-2.6.32-SLE11-SP1/include/linux/kernel.h
@@ -235,6 +235,13 @@ extern struct pid *session_of_pgrp(struc
 #define FW_WARN		"[Firmware Warn]: "
 #define FW_INFO		"[Firmware Info]: "
 
+/*
+ * HW_ERR
+ * Add this to a message for hardware errors, so that user can report
+ * it to hardware vendor instead of LKML or software vendor.
+ */
+#define HW_ERR		"[Hardware Error]: "
+
 #ifdef CONFIG_PRINTK
 asmlinkage int vprintk(const char *fmt, va_list args)
 	__attribute__ ((format (printf, 1, 0)));
